bootstrap/16-files.sh

```diff
@@ -0,0 +1,29 @@
+function install_recur() {
```

Comment:
Move the function to `functions.sh` and prepare a comment for the function. Stick to the following format:
```
# Description of the function
# Globals:
#     Global variables used and modified
# Arguments:
#     Arguments taken
# Returns:
#     Returned values other than the default exit status of the last command run
```